amethyst_renderer/src/formats/texture.rs

```diff
@@ -339,8 +339,8 @@ mod tests {
         match TextureData::from([0.25, 0.50, 0.75]) {
             TextureData::Rgba(color, _) => {
                 assert_eq!(color, [0.25, 0.50, 0.75, 1.0]);
-            },
-            _ => panic!("Expected [f32; 3] to turn into TextureData::Rgba")
+            }
```

Comment:
hum?

Reply:
Match arm removed comma at end. Took me a while to see that one too :)

Reply:
Maybe I should write more helpful messages than "hum" when I review. I was pointing out that the comma was removed, and implicitly asking if that was intended or not. I guess I'll work on communication a bit more :)

Reply:
Cargo fmt did that

Reply:
if you are using the latest version, then I assume its fine and you can ignore this comment ;)